bpf/headers/update.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Version of libbpf to fetch headers from
+LIBBPF_VERSION=0.7.0
```

Comment:
How can we make sure we fetch the correct version? libbpf comes pre-installed, or do we install it?

Reply:
Libbpf is bundled with the Cilium library. We just need the headers for compiling the C code. I'll check if there is a way that we can fetch the Cilium's libbpf version. The destination machine does not need to install any extra library, as libbpf is only used to build the eBPF VM bytecode.